network-api/networkapi/wagtailpages/pagemodels/article_fields.py

```diff
@@ -13,7 +14,7 @@
     )),
     ('card_grid', customblocks.CardGridBlock()),
     ('content', customblocks.ArticleRichText(
-        features=['bold', 'italic', 'h2', 'h3', 'h4', 'h5', 'link', 'large', 'ol', 'ul', 'image', 'hr', 'footnotes'],
+        features=base_rich_text_options + ['h2', 'h3', 'h4', 'h5', 'large', 'ol', 'ul', 'image', 'hr', 'footnotes'],
```

Comment:
does it make sense to unify this with the blog's list as one shared set that is `full_content_rich_text_options = base_rich_text_options + ['h2', 'h3', 'h4', 'h5', 'ol', 'ul', 'hr',]` and then have blog use that directly, with article using `full_content_rich_text_options + ['large', 'image', 'footnotes']`?